### ISSUE TITLE
New to HomeBridge, Phoscon, & Json coding

### ISSUE THREAD

**Issue by myitguy78** [ebaauw/homebridge-deconz#21]
I'm having issues adding the code to the HomeBridge config. Whenever I add the code, and attempt to save, I get JSON errors and doesn't save. Both HomeBridge and Phoscon is running on the same server. I'm thinking I just have a "{" or a "[" in the wrong place but no matter what I try, I can't get it to save.

**Comment by ebaauw** [comment]
Not really an issue with the plugin, but why don’t you use the _Settings_ in the _Plugins_ tab, instead of editing config.json directory?

**Comment by myitguy78** [comment]
That was the first thing I tried, but when I access the settings, I get the attached issue:
![error](https://user-images.githubusercontent.com/100226233/155195496-10ea3aa4-a398-4a5a-a3c9-25c144111e59.png)

**Comment by ebaauw** [comment]
What version of Homebridge deCONZ are you running?

**Comment by myitguy78** [comment]
HomeBridge 1.40
deCONZ 0.0.13

**Comment by myitguy78** [comment]
Or if you are referring to the deCONZ app it is 2.14.1

**Comment by myitguy78** [comment]
So I have gotten further than I was before. I was able to actually add the ConBee by going to your plugin and clicking on the wrench and then JSON Config. This added the device into the HomeBridge Config (maybe I missed this step?) Now the logs are asking to press a button:
_[2/22/2022, 2:49:47 PM] [deCONZ] Phoscon-GW: warning: request 5: api error 101: link button not pressed
[2/22/2022, 2:49:47 PM] [deCONZ] Phoscon-GW: unlock gateway to obtain API key - retrying in 15s_
Not sure where this button would be. I did see in your instructions that it would eventually connect, but as of 30min into this, it has not, and the error message is still cycling through the logs.
